docs/make.jl

```diff
@@ -33,13 +33,16 @@ makedocs(
             "Markdown"                => Any[
                 "Markdown Backend"    => "man/markdown/markdown_backend.md",
                 "Examples"            => "man/markdown/markdown_examples.md",
+            ],
+            "Markdown"                => Any[
+                "Markdown Backend"    => "man/typst/typst_backend.md",
             ]
         ],
         "Library"                     => "lib/library.md",
     ]
 )
 
 deploydocs(
-    repo = "github.com/ronisbr/PrettyTables.jl.git",
+    repo = "github.com/gustavohtc23-sudo/PrettyTables.jl.git",
```

Comment:
Please, remove this modification.
```suggestion
    repo = "github.com/ronisbr/PrettyTables.jl.git",
```